tests/lyra.edge.test.ts

```diff
@@ -34,8 +35,8 @@ t.test("Edge getters", t => {
     const nameIndex = index["name"];
 
     // Remember that tokenizers an stemmers sets content to lowercase
-    t.ok(trieContains(nameIndex, "john"));
-    t.ok(trieContains(nameIndex, "jane"));
+    t.ok(trieContains(nameIndex as unknown as Node, "john"));
+    t.ok(trieContains(nameIndex as unknown as Node, "jane"));
```

Comment:
I'm not sure why I have to cast this type. My eyes are burning, may take a second look at this later on today

Reply:
Fixed